server/controller/officeController.js

```diff
@@ -1,10 +1,10 @@
-import officesDb from '../models/officeModels';
+import officesDb from '../model/officeModel';
 
 /**
  * Class representing OfficeController
  * @class OfficeController
  */
-class OfficesController {
+export default class OfficesController {
```

Comment:
Please your class name should be singular; OfficeController. Also your controller file should be in Pascal casing. N:B the controller filename and the class name should be same based on the AirBnB guideline.

Reply:
Acknowledged.
I have implemented the feedback.